addons/crm/__init__.py

```diff
@@ -8,3 +8,8 @@
 
 from odoo import api, SUPERUSER_ID
 
+def _auto_install_iap_lead_startup(cr, registry):
```

Comment:
What is the purpose of this code ?

Reply:
Fabien asked that when we install the crm, by default lead enrichment is active and installed.
I first tried using autoinstall. But then it was not possible to uninstall the module anymore. This because
crm_iap_lead_enrichment depends on crm_iap_lead who depends on crm. I had put autoinstall on crm_iap_lead and crm_iap_lead_enrichment. But then when uninstalling crm_iap_lead_enrichment it would reinstall immediately as  crm_iap_lead was still present and can't be removed. Is there another/better way?

Reply:
Remove all that crap and make them not auto install. This can be done in custom on saas if necessary.